### PR TITLE
Updated test to clean up the collections it created

### PR DESCRIPTION
**Pull request by chartjes**
This update cleans up collections created by the test that attempts to verify that the process of reviewing content placed in specific collections works as expected.

### CODE REVIEW

**Comment by chartjes** [comment]
@leplatrem I added deleting that content to the test as well.

File: config-tests/test_remote_settings.py [hunk 1]
```diff
@@ -75,3 +75,20 @@ def test_create_collection_in_main_bucket(conf, env):
...
+
+    # Now clean up all the records we created
+    dc_response = client.delete_collection(id=collection, bucket='main-workspace')
+    assert dc_response['deleted'] is True
+
+    dc_response = client.delete_collection(id=collection, bucket='main-preview')
+    assert dc_response['deleted'] is True
+
+    dc_response = client.delete_collection(id=collection, bucket='main')
+    assert dc_response['deleted'] is True
```
Comment (leplatrem) [inline config-tests/test_remote_settings.py]: nitpick: could have used a loop `for bid in ('main-workspace', 'main-preview', 'main'):`

File: config-tests/test_remote_settings.py [hunk 2]
```diff
@@ -75,3 +75,20 @@ def test_create_collection_in_main_bucket(conf, env):
...
+
+    editor_group = '{0}-editors'.format(collection)
```
Comment (leplatrem) [inline config-tests/test_remote_settings.py]: nitpick: could have been `editors_group`

File: config-tests/test_remote_settings.py (hunk 1 above)
Comment (chartjes) [reply config-tests/test_remote_settings.py]: Not using a loop is a holdover from my PHP days where it was considered a "test smell" to have loops and conditional statements inside your tests. I will update it

**Comment by chartjes** [comment]
@leplatrem The test is now using a loop to reduce duplication and I changed the variable to 'editors_groups' at your suggestion

**Pull request merged**